stripe/src/main/java/com/stripe/android/StripeResponse.kt

```diff
@@ -46,4 +54,8 @@ internal data class StripeResponse internal constructor(
     override fun toString(): String {
         return "Request-Id: $requestId, Status Code: $responseCode"
     }
+
+    private companion object {
+        private val REQUEST_ID_HEADER_KEYS = setOf("Request-Id", "request-id")
```

Comment:
header fields are case insensitive (the values aren't though) - so we should be converting to lower case in any case
https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive